app/views/games/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="col-xs-10 col-xs-offset-1">
+
+  <div class="game-container">
+
+    <table id="board" cellpadding="0" cellspacing="0">
+      <% 7.downto(0) do |row| %>
+        <tr>
+          <% 0.upto(7) do |col| %>
+            <td>
+              

```

Comment:
Nice use of Ruby to construct the table. This wouldn't have crossed my mind and I can see how your css fits in nicely.

Is there a way to view what this would look like before merging without having to copy/paste into my local files?